eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -6,4 +6,7 @@
     <UsagePattern IdentityGlob="Microsoft.Internal.TestPlatform.Remote/17.6.33617.297" />
     <UsagePattern IdentityGlob="Microsoft.Extensions.DependencyModel/3.0.0-preview4-27615-11" />
   </IgnorePatterns>
+  <Usages>
```

Comment:
The `<UsagePattern IdentityGlob="Microsoft.Internal.TestPlatform.Remote/17.6.33617.297" />` line should be removed as part of this PR since it is addressing the prebuilt.